pkg/pool-aclamm/contracts/lib/AclAmmMath.sol

```diff
@@ -112,6 +126,20 @@ library AclAmmMath {
             }
 
             changed = true;
+        } else if (isInRange && currentTime > sqrtQ0State.startTime && currentTime <= sqrtQ0State.endTime) {
```

Comment:
```suggestion
        } else if (currentTime > sqrtQ0State.startTime && currentTime <= sqrtQ0State.endTime) {
```

There's no need to check the inRange variable here, it's already checked in the line 107